src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/ModuleFucker.kt

```diff
@@ -117,17 +121,27 @@ object ModuleFucker : Module("Fucker", Category.WORLD) {
             if (!blockPos.getState()!!.isAir) {
                 val direction = rayTraceResult.side
 
-                if (mc.interactionManager!!.updateBlockBreakingProgress(blockPos, direction)) {
-                    if (visualSwing) {
-                        player.swingHand(Hand.MAIN_HAND)
-                    } else {
-                        network.sendPacket(HandSwingC2SPacket(Hand.MAIN_HAND))
+                if (instant) {
+                    mc.networkHandler?.sendPacket(PlayerActionC2SPacket(PlayerActionC2SPacket.Action.START_DESTROY_BLOCK, blockPos, Direction.DOWN))
+                    swingHand()
```

Comment:
I highly recommend you replace the `Direction.DOWN` usages from line 124 to line 128 with the [direction](https://github.com/CCBlueX/LiquidBounce/blob/0f4fd11ae6be6fd44727b0f270559b1dd14ae54f/src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/ModuleFucker.kt#L122) variable.